applications/ContactStructuralMechanicsApplication/custom_utilities/contact_utilities.h

```diff
@@ -17,10 +17,7 @@
 // External includes
 
 // Project includes
-#include "utilities/openmp_utils.h"
-#include "utilities/math_utils.h"
 #include "contact_structural_mechanics_application_variables.h"
-#include "includes/model_part.h"
```

Comment:
this is used in this file, I wouldn't remove

Reply:
Done, reapproval?